### PR TITLE
docs: put debug arg before install arg

### PR DESCRIPTION
**Pull request by becker33**
Fixes docs to match appropriate syntax from CLI

### CODE REVIEW

File: lib/spack/docs/basic_usage.rst [hunk 1]
```diff
@@ -151,8 +150,8 @@ to get increasingly detailed output.
 The last line, with the ``[+]``, indicates where the package is
 installed.
 
-Add the debug option -- ``spack  install -d mpileaks`` -- to get additional
-output.
+Add the Spack debug option (one or more times) -- ``spack -d install
+mpileaks`` -- to get additional (and even more verbose) output.
```
Comment (nightlark) [inline lib/spack/docs/basic_usage.rst:154]: Does each additional `-d` added increase how verbose the output is?

Comment (becker33) [reply lib/spack/docs/basic_usage.rst:154]: Up to 3

**Review by nightlark — APPROVED** [review]

File: lib/spack/docs/basic_usage.rst (hunk 1 above)
Comment (tldahlgren) [reply lib/spack/docs/basic_usage.rst:154]: Or you can simply use `-ddd` for full debug output.

**Review by tldahlgren — APPROVED** [review]
LGTM

**Pull request merged**